base/strings/types.jl

```diff
@@ -160,9 +160,9 @@ function repeat(s::AbstractString, r::Integer)
 end
 
 """
-    ^(s::AbstractString, n::Integer)
+    ^(s::Union{AbstractString,Char}, n::Integer)
 
-Repeat `n` times the string `s`.
+Repeat `n` times the string or character `s`.
```

Comment:
Almost reads like "repeat n * string". Perhaps:
``Repeat the string or character `s` `n` times.``